### PR TITLE
Beta

### PR DESCRIPTION
**Pull request by adeebabdulsalam**
Deployed to polygon

### CODE REVIEW

File: artifacts/contracts/Campaign/1744wa/ClaimReferral.sol/ICollectible.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
+{
```
Comment (vizsatiz) [inline artifacts/contracts/Campaign/1744wa/ClaimReferral.sol/ICollectible.json:1]: I dont think you should keep the artifacts folder in the repository, we can generate when required locally right.
Using hardhat ?

File: scripts/deploys/deploy_referral.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    const code = await ethers.getContractFactory("contracts/Campaign/1744wa/ClaimReferral.sol:ClaimReferral")
+    const claim = await code.deploy([forwarder, rootHash]) 
+    await claim.deployed()
+    console.log(`ClaimReferral: contract deployed at: ${claim.address}`)
+    console.log(`ClaimReferral: forwarder set as: ${forwarder}`)   
+}
+
+function getForwarderBychain(chain: string): string{
+    if(chain=="polygon"){
+        return "0x5A573F05863686546d25271866de370FE6ee1B16"
```
Comment (vizsatiz) [inline scripts/deploys/deploy_referral.ts:11]: We should move these contract addresses to env

File: artifacts/contracts/Campaign/1744wa/ClaimReferral.sol/ICollectible.json (hunk 1 above)
Comment (adeebabdulsalam) [reply artifacts/contracts/Campaign/1744wa/ClaimReferral.sol/ICollectible.json:1]: artifact folder contains the build info
build info is used for contract verification on etherscan

File: scripts/deploys/deploy_referral.ts (hunk 2 above)
Comment (adeebabdulsalam) [reply scripts/deploys/deploy_referral.ts:11]: deployment scripts are mostly edited until deploying to production. After that it's never used. So I didn't think it's necessary..and we don't have any fixed environment. Sometimes I deploy from lap and sometimes from my workstation

**Pull request merged**